fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseFactory.java

```diff
@@ -50,6 +50,7 @@
      * {@link com.apple.foundationdb.record.provider.foundationdb.keyspace.LocatableResolver} retrieval requests.
      */
     public static final int DEFAULT_DIRECTORY_CACHE_SIZE = 5000;
+    private static final int API_VERSION = 600;
```

Comment:
I left this as `private`, but maybe it should be `public` so that people can inspect the `API_VERSION`? They can also do this by calling `FDB.instance().getAPIVersion()` any time after `initFdb` has been called.